packages/experimental/RadioGroup/src/RadioGroup/useRadioGroup.ts

```diff
@@ -24,6 +25,8 @@ export const useRadioGroup = (props: RadioGroupProps): RadioGroupInfo => {
     value: data.selectedKey,
     required: required || false,
     disabled: disabled || false,
+    layout: layout || 'vertical',
```

Comment:
Why are both of these needed on the state?

Reply:
Moved the logic to lookup function, but keeping layout in the state to maintain pattern of using state to fill out context.